storage/src/adb/current.rs

```diff
@@ -0,0 +1,562 @@
+//! An authenticated database (ADB) that provides succinct proofs of _any_ value ever associated
+//! with a key, and also whether that value is the _current_ value associated with it. Its
+//! implementation is based on an [Any] authenticated database combined with an authenticated
+//! [Bitmap] over the activity status of each operation.
+
+use crate::{
+    adb::{
+        any::{Any, Config as AConfig, UpdateResult},
+        Error,
+    },
+    index::{Index, Translator},
+    mmr::{
+        bitmap::Bitmap,
+        iterator::{leaf_num_to_pos, leaf_pos_to_num},
+    },
+};
+use commonware_codec::FixedSize;
+use commonware_cryptography::Hasher as CHasher;
+use commonware_runtime::{Clock, Metrics, Storage as RStorage};
+use commonware_utils::Array;
+use tracing::warn;
+
+/// Configuration for a `Current` authenticated db.
```

Comment:
```suggestion
/// Configuration for a [Current] authenticated db.
```

This will give `[Current]` hover-over help text and the ability to `CMD + click` to follow the link to the `Current` type. 

Same comment for other instances of backtick quoting types.

I realize that we use backtick quotes extensively elsewhere in the code, but I think we should probably move toward bracket quotes. (Or whatever `[This]` is called.)

Reply:
changed this and several others over to [] from ``